transforms/code/license_check/python/README.md

```diff
@@ -0,0 +1,86 @@
+# License Check 
+
+Please see the set of
+[transform project conventions](../../../README.md#transform-project-conventions)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary 
+
+This filter scans the license column of an input dataset and appends the `license_status` column to the dataset.
```

Comment:
Can you be more specific and say it selects only the specified licenses and filters out the other licenses. (same comment as on the top level readme)